src/main/webapp/style.css

```diff
@@ -117,6 +117,47 @@ button:hover {
   padding-top: 20px;
 }
 
+.userInput {
+  /*margin-top: 10px;*/
+  margin: 5px 10px 5px 10px;
+  border: 1px solid transparent;
+  border-radius: 2px 0 0 2px;
+  box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  height: 32px;
+  outline: none;
+  box-shadow: 0 2px 4px rgb(0 0 0 / 20%), 0 -1px 0px rgb(0 0 0 / 2%);
+}
+
+#origin-input,
+#destination-input, 
+#location-search-bar {
+  background-color: #fff;
+  font-family: Roboto;
+  font-size: 14px;
+  font-weight: 300;
+  margin-left: 10px;
+  padding: 0 11px 0 13px;
+  text-overflow: ellipsis;
+  width: 200px;
+}
+
+/* When clicked, border turns blue to tell user is typing in the selected input box. */
+/* Note to self: lookings kinda ugly only works on location serach bar, havent detected reason yet*/
```

Comment:
Maybe you could change the colour so that it is still obvious for accessibility, but is less ugly?

Reply:
the colour is the google blue, I think strategy wise for production, i might remove this feature as its usage and priority isnt that high, and it is more of an additional pretty/ accessibility assistance here, since if you've clicked on. the box you know youre clicking an inputing in the box. I will play around with it after finishing my high priority tasks, and if i dont get to it i think it is okay to remove it.

Reply:
I would go against removing it, because some people who use the tab key to navigate around pages would probably benefit from seeing what they've selected